Source/Blazorise/Components/Bar/Bar.razor.cs

```diff
@@ -174,10 +175,50 @@ private async void OnLocationChanged( object sender, LocationChangedEventArgs ar
             await Toggle();
     }
 
+
+    internal async Task HideOtherBarItems( BarItem barItem )
```

Comment:
Rename it to `HideAllExcept`? And make it public?

Reply:
Not, sure, seems like a very niche helper method. Do you think it's worth being public?

Reply:
Not sure haha